HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -48,6 +48,21 @@ def customiseForOffline(process):
 
     return process
 
+def customiseHLTFor46647(process):
+    for prod in producers_by_type(process, 'CtfSpecialSeedGenerator'):
+        if hasattr(prod, "DontCountDetsAboveNClusters"):
+            value = prod.DontCountDetsAboveNClusters.value()
+            delattr(prod, "DontCountDetsAboveNClusters")
+            # Replace it with cms.uint32
+            prod.DontCountDetsAboveNClusters = cms.uint32(value)
+
+        for prod in producers_by_type(process, 'SeedCombiner'):
```

Comment:
This for loop is inside the for loop from above, is this really the intention?

Reply:
no.